### ISSUE TITLE
add oredictinary to gasoline from Pnuematicraft and immersive petrolum

### ISSUE THREAD

**Issue by PandaBoy444** [PandaBoy444/ProjectGear#34]

**Comment by PandaBoy444** [comment]
There is no oredict in liquids so we'll need to manually add support for them in each mod

**Comment by TheDeviantCrafter** [comment]
It may be easier to rename them and add a conversion recipe of some sort. Pneumaticraft likes technical names and acronyms, so I'm sure there's a term we can use.

**Comment by PandaBoy444** [comment]
I'd prefer removing one and adding all the it's functionality to the other

**Comment by PandaBoy444** [comment]
Harder but much cleaner

**Comment by TheDeviantCrafter** [comment]
Agreed, but in case that's not possible I wanted to float an alternative.

**Comment by PandaBoy444** [comment]
[motorboat fuel api](http://crafttweaker.readthedocs.io/en/latest/#Mods/Immersive_Petroleum/CraftTweaker_Support/FuelRegistration/)

**Comment by almightyfoon** [comment]
added support for PC gasoline to the motor boat and portable generator for Immersive petroleum. Still need to make sure that the generation rates are the same.

**Comment by PandaBoy444** [comment]
How is the status on this?

**Comment by TheDeviantCrafter** [comment]
I don't think anyone got very far

**Comment by boblovespi** [comment]
It might be easier to wait until oil/gas/fuel processing is complete on the diagram to prevent wasted effort, and also because new fluids might be made, or new processes

**Comment by TheDeviantCrafter** [comment]
What if we just add fluid converters for now? https://minecraft.curseforge.com/projects/fluidconverters

**Comment by boblovespi** [comment]
I'm fine with it, just keep in mind it will change balance and also make it clear that the mod and whatever it adds is temporary
